2016/playbook/session_3/experiments/improve_social_behavior_of_hackers_with_card_games.md

```diff
@@ -0,0 +1,17 @@
+# Running card games during lunch
+
+## Implementation
+
+-Run a card game during lunch which initiates conversation and increases hacker productivity after lunch
```

Comment:
There should be a space between the dash and the first word in an unordered list.